ImageProcessing/README.md

```diff
@@ -0,0 +1,747 @@
+# Optional Module: Coordinate a serverless image processing workflow with AWS Step Functions
```

Comment:
good catch it was from previous iterations of the instructions. just removed them